docs/core/tools/dotnet-publish.md

```diff
@@ -228,7 +228,7 @@ For more information, see the following resources:
 - [.NET Core application publishing overview](../deploying/index.md)
 - [Publish .NET Core apps with the .NET Core CLI](../deploying/deploy-with-cli.md)
 - [Target frameworks](../../standard/frameworks.md)
-- [Runtime IDentifier (RID) catalog](../rid-catalog.md)
+- [Runtime Identifier (RID) catalog](../rid-catalog.md)
```

Comment:
I found two other occurrences of this. Opened https://github.com/dotnet/docs/pull/20136 for that.